addons/product/models/product_template.py

```diff
@@ -149,6 +149,18 @@ def _get_default_volume_uom(self):
 
     item_ids = fields.One2many('product.pricelist.item', 'product_tmpl_id', 'Pricelist Items')
 
+    has_configurable_attributes = fields.Boolean("Is a configurable product", compute='_compute_has_configurable_attributes', store=True)
+
+    @api.multi
+    @api.depends('attribute_line_ids')
```

Comment:
If this is going to be stored (why?) this depends is not correct. It must at least include `attribute_line_id.value_ids` (which can change) and I'm not even sure this will be enough.

Reply:
I only move it from sale_product_configurator, but in my mind, it is stored because it is used as a related on sale order lines to know whether the line is configurable by the product configurator.
(About the depends, I fixed it)